### DISCUSSION TITLE
140

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#74] [General]
# 潮流周刊第140期 - 爱王小波

封面图来源于周末去了一趟「我的精神家园 · 王小波书店」，在杭州武林附近，很安静的一个地方，从大学开始读王小波，这种睿智、理性、有趣又特立独行的人我可太喜欢了。 

https://weekly.tw93.fun/posts/140-%E7%88%B1%E7%8E%8B%E5%B0%8F%E6%B3%A2

**Comment by thetbw** [comment]
我们这附近图书馆是太舒服了，有空调wifi,座位有阅读灯，不看书整理整理一周的事情，也是个不错的选择

**Reply by tw93** [reply]
的确，图书馆是一个很好的地方

**Comment by zimiyu** [comment]
Maple Bookmarks真的太太太太棒了，谢谢大佬

**Reply by tw93** [reply]
谢谢喜欢，有什么需求尽管提

**Reply by 55utah** [reply]
提小需求
1. 搜出来的书签，鼠标放上去可以hover出全部标题内容吗 ～ 
2. 能否通过快捷键替代点击，比如command + e召唤出来后，再搜索，最后看到内容，可以直接 command + 1选择第一个

**Reply by 55utah** [reply]
哦 又想到一个 搜索出来的结果默认聚焦第一个 回车就访问 多个时 可以方向键选择 总之就是可以不用鼠标

**Reply by tw93** [reply]
安排，这两个都很有道理

**Comment by zhangtianqi93** [comment]
顶部状态栏的日期时间是什么软件呀？
